tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -225,6 +226,12 @@ def check_read_nwb(self, nwbfile_path: str):  # This is currently structured to
 
                 assert spatial_series.timestamps[0] == 0.0
 
+    # TODO: undo this skip in future PR
```

Comment:
Why are we skipping this? is this because I pre-wrapped in DataIO for this one?

Reply:
I did not look super close at this special case since there was already enough here. Maybe you're more familiar and can disable in a splinter branch of this to see what goes wrong

Reply:
OK. I will take a look.